python/nano/src/bigdl/nano/tf/keras/__init__.py

```diff
@@ -13,7 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import tensorflow as tf
 
-
+try:
+    from ..quantization import quantize
+    tf.keras.Model.quantize = quantize
+except ImportError:
+    print("Warning: Intel Neural Compressor should be installed if you need quantization.")
```

Comment:
Maybe adding how to install INC in the message？

Reply:
If a user does not install INC, it will always get this warning, right? I think a better behavior is raising an error when a user tries to use quantization.

Reply:
Accepted in new commits.